apps/dashboard/src/contract-ui/tabs/tokens/components/burn-button.tsx

```diff
@@ -24,27 +48,112 @@ export const TokenBurnButton: React.FC<TokenBurnButtonProps> = ({
   });
 
   const hasBalance = tokenBalanceQuery.data && tokenBalanceQuery.data > 0n;
+  const [open, setOpen] = useState(false);
+  const sendConfirmation = useSendAndConfirmTransaction();
+  const trackEvent = useTrack();
+  const form = useForm({ defaultValues: { amount: "0" } });
+  const { onSuccess, onError } = useTxNotifications(
+    "Tokens burned successfully",
+    "Failed to burn tokens",
+    contract,
+  );
```

Comment:
prefer removing this whenever u come across it and use `toast.promise()` instead

Reply:
@jnsdls do we also do `useInvalidateContractQuery` like how it's setup inside useTxNotification at the moment?

Reply:
no we should no longer need it, v5 sdk handles it for us